scripts/utils/findConflicts.js

```diff
@@ -45,14 +60,13 @@ const { argv } = yargs
     describe: 'produce debugging log',
     type: 'boolean',
   })
-  .demandOption(
-    ['config'],
-    'Please provide config file holds network gateway and DB connection URI',
-  )
+  // .demandOption(
+  //   ['config'],
+  //   'Please provide config file holds network gateway and DB connection URI',
+  // )
```

Comment:
It should be deleted but I commented it to get your confirmation
@aminlatifi

Reply:
Yes

Reply:
done